package.json

```diff
@@ -33,7 +33,7 @@
     "@babel/plugin-proposal-class-properties": "^7.13.0",
     "@babel/plugin-proposal-private-methods": "^7.13.0",
     "@types/node": "^12.20.10",
-    "electron": "^12.0.13",
+    "electron": "9.4.4",
```

Comment:
We don't want to revert to the old version yet, because we're still experimenting on fixing the performance on the newest electron versions.

Reply:
To my knowledge Null tested all versions and couldn't find a better version.

Reply:
i separately tested all major / used by other clients electron version, and found out that 9.4.4 runs the best:
![image](https://user-images.githubusercontent.com/21956756/130432282-1c043f48-df1a-423a-9f00-b965f451b3e4.png)
i tested each for around 10 minutes with the same conditions.

Reply:
the numbers are fps values + on latest electron the game freezes when i shoot and unfreezes when i don't, this would be fixed by aim freeze fix but that hinders performance drastically

Reply:
Imma agree on this one. 9.4.4 is our best option

Reply:
Then I don't see any reason not to revert to 9.4.4 for now, at least if it works with all components.

Reply:
Would it be a good idea to have 2 branches with v9 and the latest Electron version? I sometimes get feedback about how v12 is performing better for some people, and v9 is already out of official "supported versions".

Reply:
As long as there are no breaking changes, I think we should be fine with two branches.

Reply:
would both branches get all updates?

Reply:
yes. only the electron base version would be different

Reply:
or would 9.4.4 be the primary branch and v12 the test one? idk having 2 branches seems like more work, if someone wants to test v12 performance they could just change it in the package.json or use the experimental branch